src/main/java/model/Offer.java

```diff
@@ -5,7 +5,7 @@
  */
 public class Offer {
 
-    private long id;
+    private Long id;
```

Comment:
Do we really need Long here? Just to consider. Do you think we will ever reach 2,147,483,647 orders? I hope so :) but it seems to be unreal for now.